CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ endif()
 
 # Link and create SDL2 Definition if we're using SDL2
 if (SDL2_ENABLED)
+	find_package(SDL2 REQUIRED)
```

Comment:
This is something you'd need to try at your end. It works for me. If this doesn't work for you, you might need to replace `REQUIRED` with `QUIET `.